composer.json

```diff
@@ -15,6 +15,7 @@
     },
     "require": {
         "ezsystems/platform-ui-assets-bundle": "~0.2",
+        "ezsystems/repository-forms": "dev-EZP-24379_platformUIIntegration",
```

Comment:
Is it normal that you merged that to master ? Because I see that this PR is merged too ...